packages/frontend/src/pages/Model.spec.ts

```diff
@@ -37,13 +40,40 @@ vi.mock('../utils/client', async () => {
   };
 });
 
+vi.mock('/@/stores/inferenceServers', () => ({
+  inferenceServers: vi.fn(),
+}));
+
 const model: ModelInfo = {
   id: 'model1',
   name: 'Model 1',
   properties: {},
   description: '',
 };
 
+beforeEach(() => {
+  (inferenceStore.inferenceServers as unknown) = readable<InferenceServer[]>([]);
```

Comment:
suggestion but using beforeAll AFAIK
```
  Object.defineProperty(inferenceStore, 'inferenceServers', {
    value: readable<InferenceServer[]>([]);
  });
```

Reply:
I like to use beforeEach before the store is restored every time